src/main/groovy/org/broadinstitute/orsp/consent/DataUseRestrictionDTO.groovy

```diff
@@ -0,0 +1,94 @@
+package org.broadinstitute.orsp.consent
+
+import org.broadinstitute.orsp.utils.IssueUtils
+import org.grails.web.json.JSONArray
+
+class DataUseRestrictionDTO {
```

Comment:
I'm not sure I understand the need for a DTO here. Why not just put the `fromParams` method into a utility class and return a new `DataUseRestriction` ? There's a ton of copy/paste going on between this class, `DataUseRestriction`, and also in the `DataUserLetterService.createSdul` method. Having to maintain all of these duplicate fields in three places is going to be painful over time. 

Note that even though you can't populate a complete DUR from the input, the vast majority of it *can* be populated. What you'd really be creating with a utility method is a partial DUR. You can finish the population in the `createSdul` method.

Reply:
Agree! I worked on that